docs/guides/high-availability/concepts-multi-site.adoc

```diff
@@ -100,17 +97,22 @@ The statement "`No data loss`" depends on the setup not being degraded from prev
 == Known limitations
 
 Site Failure::
-* A successful failover requires a setup not degraded from previous failures.
+A successful failover requires a setup not degraded from previous failures.
 All manual operations like a re-synchronization after a previous failure must be complete to prevent data loss.
 Use monitoring to ensure degradations are detected and handled in a timely manner.
 
 Out-of-sync sites::
-* The sites can become out of sync when a synchronous {jdgserver_name} request fails.
+The sites can become out of sync when a synchronous {jdgserver_name} request fails.
 This situation is currently difficult to monitor, and it would need a full manual re-sync of {jdgserver_name} to recover.
 Monitoring the number of cache entries in both sites and the {project_name} log file can show when resynch would become necessary.
 
 Manual operations::
-* Manual operations that re-synchronize the {jdgserver_name} state between the sites will issue a full state transfer which will put a stress on the system (network, CPU, Java heap in {jdgserver_name} and {project_name}).
+Manual operations that re-synchronize the {jdgserver_name} state between the sites will issue a full state transfer which will put a stress on the system (network, CPU, Java heap in {jdgserver_name} and {project_name}).
+
+Two sites restriction::
+This setup is tested and supported only with two sites.
```

Comment:
I think what we have is good, but I have reworded it below in an attempt to make it flow better. I'm not sure how well I accomplished that though, so feel free to ignore my suggestion :slightly_smiling_face: 

"This setup is tested and supported only with two sites. Each additional site increases overall latency as it is necessary for data to be synchronously written to each site. Furthermore, the probably of network failures, and therefore downtime, also increases. Therefore, we do not support more than two sites as we believe it would lead to a deployment with inferior stability and performance."

Reply:
This sounds good. The only thing that caught my attention is that your suggestion can lead to an impression more than 2 sites will never be supported. Is this true? Would the performance be so bad that we won't be able to optimize it to a usable state?

Reply:
The biggest limitation is the SYNC replication of data between sites, while that is in place performance will always suffer as we increase the number of sites. Performance could maybe be good enough to have 3 sites, but it will be worse than with 2 sites.

I know @pruivo has wanted to move to ASYNC for a long time, but there's pros and cons as we've discussed many times :slightly_smiling_face:

Reply:
More sites mean more latency and more chances of deadlocks. Also, if we want to support more sites, we must change the configuration to keep the data consistent. We need a 2PC configuration between the 3 sites :disappointed:

Reply:
Thank you @ryanemerson and @pruivo, based on your responses I updated the doc to the suggested one. I think it makes sense to be more strict with the restriction to 2 sites in the docs.